tenets/codelingo/go/naked-chan-access/codelingo.yaml

```diff
@@ -0,0 +1,16 @@
+tenets:
+  - name: naked-chan-access
+    actions:
+      codelingo/review:
+        comment: Reading from a channel without wrapping it in a select statement is unsafe. 
+    query: |
+      import codelingo/ast/go
+
+      go.func_decl(depth = any):
+        exclude:
+          go.select_stmt(depth = any):
```

Comment:
might be better to use `comm_clause` here? *might* work better to catch naked channel usage inside the branches